src/Symfony/Component/ErrorHandler/DebugClassLoader.php

```diff
@@ -170,7 +165,7 @@ public function loadClass($class)
         $this->checkClass($class, $file);
     }
 
-    private function checkClass($class, $file = null)
+    private function checkClass(string $class, $file = null): void
```

Comment:
`string $file = null`? I know it could be `false' (a few lines above) but I think it's okay again, probably changing it to `$file = ''` would make more sense now.

Reply:
good pointed, changed to empty string, if statements are also fine with an empty string instead of false